### DISCUSSION TITLE
how to get id of line using Tables\Actions\Action::make()

### DISCUSSION THREAD

**Discussion by phamduc1823** [filamentphp/filament#8041] [General]
is there a way to get the id of that row in Tables\Actions\Action::make().
Example id: there is a list table with ids 1, 2, 3, 4 I want Tables\Actions\Action::make() to get id = 1 in the table to continue custom work

**Comment by pboivin** [comment]
You can access the entire record like this:
```php
->action(function ($record) {
    // use $record->id
})
```

**Reply by phamduc1823** [reply]
thank you very much
